### DISCUSSION TITLE
【报错】不支持外链的纹理贴图

### DISCUSSION THREAD

**Discussion by terrykingcha** [galacean/discussion#79] [Editor]
![image](https://github.com/galacean/discussion/assets/1314662/06665716-fd40-4b6a-8218-b6a744a40022)

**Comment by gz65555** [comment]
目前使用的 https://github.com/donmccurdy/glTF-Transform/issues/207 在 glb 里是不支持外链图片的

**Reply by terrykingcha** [reply]
考虑优化下吗？不然只能自己写个gltf-loader了

**Reply by gz65555** [reply]
这个暂时没有计划，因为 glb 本身外链图片就很奇怪，如果外链图片建议转成 gltf
